osu.Game.Rulesets.Mania/Mods/ManiaModMirror.cs

```diff
@@ -15,6 +15,7 @@ public class ManiaModMirror : Mod, IApplicableToBeatmap
         public override string Name => "Mirror";
         public override string Acronym => "MR";
         public override ModType Type => ModType.Conversion;
+        public override string Description => "Notes are flipped horizontally";
```

Comment:
period for consistency
```suggestion
        public override string Description => "Notes are flipped horizontally.";
```